pyaedt/edb.py

```diff
@@ -488,7 +489,7 @@ def import_layout_pcb(
     ):
         """Import a board file and generate an ``edb.def`` file in the working directory.
 
-        This function supports all AEDT formats (DXF, GDS, SML (IPC2581), BRD, TGZ ...).
+        This function supports all AEDT formats (DXF, GDS, SML (ipc2581), BRD, TGZ ...).
```

Comment:
```suggestion
        This function supports all AEDT formats, including DXF, GDS, SML (IPC2581), BRD, and TGZ.
```
Style guide says not to use ellipses.